dpnp/tests/test_linalg.py

```diff
@@ -2037,6 +2042,23 @@ def _apply_pivots_rows(A_dp, piv_dp):
         rows = dpnp.asarray(rows)
         return A_dp[rows]
 
+    @staticmethod
+    def _make_nonsingular_nd_np(shape, dtype, order):
+        A = generate_random_numpy_array(shape, dtype, order)
+        m, n = shape[-2], shape[-1]
+        k = min(m, n)
+        A3 = A.reshape((-1, m, n))
+        for B in A3:
+            for i in range(k):
+                off = numpy.sum(numpy.abs(B[i, :n])) - numpy.abs(B[i, i])
+                B[i, i] = A.dtype.type(off + 1.0)
+
+        A = A3.reshape(shape)
```

Comment:
Based on NumPy [documentation](https://numpy.org/doc/2.2/reference/generated/numpy.reshape.html#numpy.reshape):
> there is no guarantee of the memory layout (C- or Fortran- contiguous) of the returned array

Reply:
Fix that
Thank you!